thoth/prescriptions_refresh/handlers/package_downloads.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python3
+# thoth-prescriptions-refresh
+# Copyright(C) 2021 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Retrieve the number of downloads from PyPI for a given package"""
+
+
+from datetime import datetime
+from google.cloud import bigquery
+import logging
+import matplotlib.pyplot as plt
+import os
+from typing import TYPE_CHECKING
+from typing import Any
+from typing import Dict
+from typing import Optional
+from typing import Tuple
+
+if TYPE_CHECKING:
+    from thoth.prescriptions_refresh.prescriptions import Prescriptions
+
+_LOGGER = logging.getLogger(__name__)
+
+_PYPI_POPULARITY_LOW = int(os.getenv("THOTH_PRESCRIPTIONS_REFRESH_PYPI_POPULARITY_LOW", 20))
+_PYPI_POPULARITY_MODERATE = int(os.getenv("THOTH_PRESCRIPTIONS_REFRESH_PYPI_POPULARITY_MODERATE", 100))
+_PYPI_POPULARITY_HIGH = int(os.getenv("THOTH_PRESCRIPTIONS_REFRESH_PYPI_POPULARITY_HIGH", 1000))
+
+_PACKAGE_DOWNLOADS_PRESCRIPTION_NAME = "package_downloads.yaml"
+_PACKAGE_DOWNLOADS_PRESCRIPTION_CONTENT = """\
+units:
+  wraps:
+  - name: {prescription_name}
+    type: wrap
+    should_include:
+      adviser_pipeline: true
+    match:
+      state:
+        resolved_dependencies:
+        - name: {package_name}
+    run:
+      justification:
+      - type: INFO
+        message: |
```

Comment:
This can cause serialization issues on the client-side. Could the message be a single-line message? Eventually, you can provide multiple justifications inside the `justification` listing.